### ISSUE TITLE
Content view for views is broken for views created with no schema binding

### ISSUE THREAD

**Issue by DanWilliamsAu** [jakob/Postico#578]
The 'content' view does not work for redshift views created WITH NO SCHEMA BINDING (https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_VIEW.html)

## To reproduce

Create a view with no schema binding 

```sql
CREATE view test AS
SELECT 1 as test_column
WITH NO SCHEMA BINDING;     
```

Then try to view the 'content' tab of the view, you get and error message

>Could not load table rows.
>Query failed
>PostgreSQL said: syntax error at or near "FROM"

It looks like postico generates an SQL statement to get the content view, but with the no schema binding option it cannot generate it correctly. The resulting query from clicking content looks like
```sql
SELECT  FROM "test"    LIMIT 1000;
```

which is obviously a syntax error!

## What software versions are you using?

Postico version:  1.3.2

macOS version:  10.13.6

PostgreSQL version:  PostgreSQL 8.0.2 on i686-pc-linux-gnu, compiled by GCC gcc (GCC) 3.4.2 20041017 (Red Hat 3.4.2-6.fc3), Redshift 1.0.2915

Thanks very much in advance for looking at this issue, and thanks for creating postico!

**Comment by jakob** [comment]
This was fixed in Postico v1.3.3

See also #493

Can you download the latest version of Postico and try again?

**Comment by DanWilliamsAu** [comment]
🤦‍♂️ yep, downloading the latest version and this bug is fixed! Thanks and sorry for reporting an old bug!
